ansible/roles/test/files/tools/loganalyzer/loganalyzer_common_ignore.txt

```diff
@@ -27,6 +27,7 @@ r, ".* INFO kernel:.*"
 r, ".* INFO systemd.*"
 r, ".* ERR kernel:.* Module gpio_ich is blacklisted.*"
 r, ".* skipping since it causes crash: SAI_STP_ATTR_BRIDGE_ID.*"
+r, ".*y_cable_port.*GET http.*"
```

Comment:
If only mvrf/test_mgmtvrf.py needs to ignore this error msg, I think you should skip in mvrf/test_mgmtvrf.py